pandas/compat/pickle_compat.py

```diff
@@ -64,7 +64,7 @@
 # our Unpickler sub-class to override methods and some dispatcher
 # functions for compat and uses a non-public class of the pickle module.
 class Unpickler(pickle._Unpickler):
-    def find_class(self, module, name):
+    def find_class(self, module, name):  # type: ignore[no-untyped-def]
```

Comment:
[Based on typeshed](https://github.com/python/typeshed/blob/24d39de94be193e121c9fe10f11ab6d86110c627/stdlib/pickle.pyi#L183C5-L183C76), you can probably do:
`def find_class(self, module: str, name: str) -> Any: ...`